src/Arcus.Scripting.Tests.Integration/Arcus.Scripting.DevOps.tests.ps1

```diff
@@ -33,5 +33,40 @@ InModuleScope Arcus.Scripting.DevOps {
                 }
             }
         }
+        Context "Set ARM outputs as DevOps variable" {
+            It "Adds ARM outputs as a new default DevOps variable to an existing variable group" {
+                # Arrange
+                $existingVariableGroup = $config.Arcus.DevOps.VariableGroup.Name
+                $variableName = "MyVariable"
+                $variableValue = [System.Guid]::NewGuid()
+                $env:ArmOutputs = "{ ""$variableName"": [ { ""Name"": ""$variableName"", ""Value"": { ""value"": ""$variableValue"" } } ] }"
+
+                $project = "$env:SYSTEM_TEAMPROJECT"
+                $projectUri = "$env:SYSTEM_TEAMFOUNDATIONCOLLECTIONURI"
+                $apiVersion = "4.1-preview.1"
+                
+                $headers = @{ Authorization = "Bearer $env:SYSTEM_ACCESSTOKEN" }
+                $variableGroup = $null
+
+                #try {
+                    # Act
+                    Set-AzDevOpsArmOutputsToVariableGroup -VariableGroupName $existingVariableGroup
+
+                    # Assert
+                    $getVariableGroupUrl= $projectUri + $project + "/_apis/distributedtask/variablegroups?api-version=" + $apiVersion + "&groupName=" + $existingVariableGroup
+                    $variableGroup = Invoke-RestMethod -Uri $getVariableGroupUrl -Headers $headers -Verbose
+                    $variableGroup = $variableGroup.value[0]
+                    $variableGroup.variables.$variableName | Should -Not -Be $null
+                    $variableGroup.variables.$variableName.value | Should -Be "@{value=$variableValue}"
+                #} finally {
```

Comment:
Why is the try/finally construct commented out ?

Reply:
Because I'm still working on it. This PR is not ready for a review.